jetson/odrive_bridge/src/__main__.py

```diff
@@ -95,7 +95,7 @@ class State(object):
     def __init__(self):
         print('Processing current state:', str(self))
 
-    def on_event(self, event):
+    def state_on_event(self, odrive_event):

```

Comment:
i think i see what you're doing here..trying to make the code clearer w more verbose variable names but it's getting really wordy..would suggest just adding better comments next time

Reply:
ya